### PR TITLE
Validate TLS endpoints

### PR DESCRIPTION
**Pull request by bissenbay**
Fixes: https://github.com/thoth-station/integration-tests/issues/33

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -26,13 +27,13 @@ Run integration tests against stage deployment:
 
 .. code-block:: console
 
-  THOTH_USER_API_HOST=stage.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.stage.thoth-station.ninja ./test.sh
+  THOTH_USER_API_HOST=stage.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.stage.thoth-station.ninja THOTH_AMUN_API_HOST=amun.stage.thoth-station.ninja ./test.sh
```
Comment (bissenbay) [inline README.rst:30]: Do we have such endpoint like `amun.stage.thoth-station.ninja`?

File: README.rst [hunk 2]
```diff
@@ -26,13 +27,13 @@ Run integration tests against stage deployment:
...
 
 Run integration tests against test deployment (default behaviour):
 
 .. code-block:: console
 
-  THOTH_USER_API_HOST=test.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.test.thoth-station.ninja ./test.sh
+  THOTH_USER_API_HOST=test.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.test.thoth-station.ninja THOTH_AMUN_API_HOST=amun.test.thoth-station.ninja ./test.sh
```
Comment (bissenbay) [inline README.rst:36]: Can we make the endpoint `amun-api-thoth-test-core.apps.ocp.prod.psi.redhat.com` as `amun.test.thoth-station.ninja`?

File: README.rst (hunk 1 above)
Comment (harshad16) [reply README.rst:30]: we do , its here: https://console-openshift-console.apps.ocp.prod.psi.redhat.com/k8s/ns/thoth-amun-api-stage/routes
however we need to direct route53 to ocp cluster

File: README.rst (hunk 2 above)
Comment (harshad16) [reply README.rst:36]: https://console-openshift-console.apps.ocp.prod.psi.redhat.com/k8s/ns/thoth-test-core/routes
we need to direct route53 to ocp cluster

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :+1:

**Pull request merged**